src/app/reports/invoice/invoice.html

```diff
@@ -16,13 +16,13 @@ <h3 class="panel-title">Delivery Rounds</h3>
       </tr>
     </thead>
     <tbody>
-      <tr ng-repeat="round in reportsCtrl.deliveryRounds">
+      <tr ng-repeat="round in reportsAllCtrl.deliveryRounds">
```

Comment:
ok i ll go ahead and merge this, so the pagination can take care of it


On Mon, Aug 10, 2015 at 10:27 AM, Jideobi Benedine Ofomah <
notifications@github.com> wrote:

> In src/app/reports/invoice/invoice.html
> <https://github.com/eHealthAfrica/direct-delivery-dashboard/pull/62#discussion_r36614252>
> :
>
> > @@ -16,13 +16,13 @@ <h3 class="panel-title">Delivery Rounds</h3>
> >        </tr>
> >      </thead>
> >      <tbody>
> > -      <tr ng-repeat="round in reportsCtrl.deliveryRounds">
> > +      <tr ng-repeat="round in reportsAllCtrl.deliveryRounds">
>
> @freefony <https://github.com/freefony> noted, like i said, i just moved
> this file, and i think we already have a ticket for paginating and
> optimising this invoice table Haykel created.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/eHealthAfrica/direct-delivery-dashboard/pull/62/files#r36614252>
> .
>